RELEASE_NOTES.txt

```diff
@@ -1,6 +1,120 @@
 Apache Ignite Release Notes
 ===========================
 
+Apache Ignite In-Memory Distributed Database 2.14.0
+-----------------------------------------------------------
+
+(!) WARNINGS:
+* Removed LOCAL caches support.
+* Removed scalar module.
+
+Ignite:
+* Added CDC binary meta change events.
+* Added a snapshot create operation metrics.
+* Added a user info to the sql queries view.
+* Added cache change events for CDCCosumer.
+* Added cache metrics command for Control Script.
+* Added control.sh command that schedules index rebuild via the maintenance mode.
+* Added opportunity to specify single partition for IndexQuery.
+* Added pluggable affinity mapping function for the partition awareness usage.
+* Added snapshot status command to control.sh and JMX.
+* Added support of optional beans for SpringResource annotation injection.
+* Added system view and histogram to show the distribution of hot and cold pages in the page memory.
```

Comment:
22. Added a system view and histogram to show the distribution of hot and cold pages in the memory page.